src/Components/NavInfoList.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const NavInfoList = () => {
+  const [hiddenSearchBar, setHiddenSearchBar] = useState(true);
+  const [bellHiddenBox, setBellHiddenBox] = useState(true);
+  const [loginHiddenBox, setLoginHiddenBox] = useState(true);
+
+  const handleSearch = () => {
+    setHiddenSearchBar(hiddenSearchBar => !hiddenSearchBar);
+  };
+
+  const handleBellBox = () => {
+    setBellHiddenBox(bellHiddenBox => !bellHiddenBox);
+  };
+
+  const handleLoginBox = () => {
+    setLoginHiddenBox(loginHiddenBox => !loginHiddenBox);
+  };
+
+  return (
+    <NavRightList>
+      <NavRightLayout>
+        {hiddenSearchBar ? (
+          <button onClick={handleSearch}>
+            <NavRightIcon className="fas fa-search" />
+          </button>
+        ) : (
+          <SearchBar>
+            <SearchBarInput />
+            <SearchBarIcon className="fas fa-search" onClick={handleSearch} />
+          </SearchBar>
+        )}
+      </NavRightLayout>
+      <NavRightLayout>
+        <button onMouseEnter={handleBellBox}>
+          <NavRightIcon className="fas fa-bell" />
+        </button>
+        {bellHiddenBox ? null : (
+          <BellHiddenBox onMouseLeave={handleBellBox}>
+            최근 알림 메시지가 없습니다.
+          </BellHiddenBox>
+        )}
```

Comment:
```suggestion
        {!bellHiddenBox && (
          <BellHiddenBox onMouseLeave={handleBellBox}>
            최근 알림 메시지가 없습니다.
          </BellHiddenBox>
        )}
```
&& 연산자를 써야하는 상황으로 보입니다

Reply:
반영 완료했습니다